packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -5216,6 +5216,20 @@ export default class Meeting extends StatelessWebexPlugin {
     }
     this.lastVideoLayoutInfo = cloneDeep(layoutInfo);
 
+    this.locusInfo.once(LOCUSINFO.EVENTS.CONTROLS_MEETING_LAYOUT_UPDATED, (envelope) => {
```

Comment:
This wont happen but do we need to check if what comes back as layout is what you asked for ?